src/FieldError.php

```
@@ -116,31 +88,59 @@ static function ($value) {
+    /**
+     * Get all errors for all properties.
+     *
+     * @param array $onlyProperties List of properties to return errors.
+     * @param bool $first Whether to return only the first error of each property.
+     *
+     * @return array The errors for all properties as a one-dimensional array. Empty array is returned if no error.
+     *
+     * @phpstan-param list<string> $onlyProperties
+     * @phpstan-return array<int|string, string>
+     */
+    public function getSummary(array $onlyProperties = [], bool $first = false): array
```

PHPMD (2.15.0)

[error] The method getSummary has a boolean flag argument $first, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)